static/js/main.js

```diff
@@ -0,0 +1,217 @@
+var xBeeArray = []
+var newxBeeArray = []
+var tabletext = ''
+function begin(){
+   var body = document.getElementById('mainb')
+   var text = `<h1 id='tabTitle'>Xbee Nodes</h1>
+   <div id='cTable'></div>
+   <button onclick='refresh()'>Refresh</button>`
```

Comment:
Instead of a refresh button, you can use [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) to run the refresh routine every second or so.

Reply:
Maybe make it 5-10 seconds actually since it is unlikely that the units will update that often.